README.md

```diff
@@ -15,7 +15,7 @@ This includes
 
 Rafiki is made up of several components including an Interledger connector, a high-throughput accounting database called [TigerBeetle](https://rafiki.dev/concepts/accounting/tigerbeetle/), and several APIs:
 
-- the [Admin API](https://rafiki.dev/apis/backend/schema/) to create [peering relationships](https://rafiki.dev/concepts/interledger-protocol/peering/), add supported [assets](https://rafiki.dev/concepts/asset/), and issue [payment pointers](https://rafiki.dev/reference/glossary/#payment-pointer)
+- the [Admin API](https://rafiki.dev/apis/backend/schema/) to create [peering relationships](https://rafiki.dev/concepts/interledger-protocol/peering/), add supported [assets](https://rafiki.dev/concepts/asset/), and issue [wallet addresses](https://rafiki.dev/reference/glossary/#payment-pointer)
```

Comment:
does the article/url need to be renamed here? https://rafiki.dev/reference/glossary/#payment-pointer

Reply:
I guess `/reference/glossary#payment-pointer` is referenced many places, not just here.